RecoBTag/ONNXRuntime/interface/tensor_configs.h

```diff
@@ -28,4 +29,48 @@ namespace deepvertex {
 
 }  // namespace deepvertex
 
+namespace parT {
+
+  enum InputFeatures {
+    kChargedCandidates=0,
+    kNeutralCandidates=1,
+    kVertices=2,
+    kChargedCandidates4Vec=3,
+    kNeutralCandidates4Vec=4,
+    kVertices4Vec=5
+  };
+  
+  const std::map<unsigned int, InputFeatures> InputIndexes{
```

Comment:
as a method to loop through all the enum values, this is somewhat redundant (requires adding the same information in two places: the enum definition and this map). I checked with core software and they recommend something like the following:
```
  enum InputFeatures {
    kBegin=0,
    kChargedCandidates=kBegin,
    kNeutralCandidates=1,
    kVertices=2,
    kChargedCandidates4Vec=3,
    kNeutralCandidates4Vec=4,
    kVertices4Vec=5,
    kEnd=6
  };
  ```
  then you can just loop from `kBegin` to `kEnd`.

Reply:
okay that will help...